### PR TITLE
Bump quic-go to 0.32.0

### PR DESCRIPTION
**Pull request by tobyxdd**

### CODE REVIEW

**Comment by charlieporth1** [comment]
> 

Please provide a description so we can test accordingly

File: adminlistener.go [hunk 1]
```diff
@@ -9,8 +9,8 @@ import (
 	"net/http"
 	"time"
 
-	"github.com/lucas-clemente/quic-go"
-	"github.com/lucas-clemente/quic-go/http3"
+	"github.com/quic-go/quic-go"
```
Comment (charlieporth1) [inline adminlistener.go:12]: Why was this library chosen over the previous one

**Review by charlieporth1 — APPROVED** [review]

File: adminlistener.go (hunk 1 above)
Comment (tobyxdd) [reply adminlistener.go:12]: It's the same library, they just changed its package name.

Comment (tobyxdd) [reply adminlistener.go:12]: See https://github.com/quic-go/quic-go/issues/3611 for more information

**Pull request merged**